### PR TITLE
Bump to `0.3.0a1`

### PR DESCRIPTION
**Pull request by jtpio**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by jtpio** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% set version = "0.3.0a0" %}
+{% set version = "0.3.0a1" %}
 {% set js_version = version.replace("a", "-alpha.").replace("b", "-beta.").replace("rc", "-rc.") %}
 {% set jupyterlab_version = ">=4.1.1,<4.2" %}
```
Comment (bollwyvl) [inline recipe/meta.yaml:3]: do we need to bump this?

**Comment by bollwyvl** [comment]
Put differently: are there _new_ breaking changes?

**Comment by jtpio** [comment]
> Put differently: are there _new_ breaking changes?

In `0.3.0a1` compared to `0.3.0a0`? None that are known.

**Comment by bollwyvl** [comment]
Great! Lgtm, will merge when back at keys

**Pull request merged**